packages/pds-reachability/pds-reachability.0.2.3/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+synopsis: "A PDS reachability query library"
+description:
+  "This library performs efficient reachability queries on abstractly specified push-down systems."
+maintainer: "JHU PL Lab <pl.cs@jhu.edu>"
+authors: "JHU PL Lab <pl.cs@jhu.edu>"
+license: "Apache-2.0"
+homepage: "https://github.com/JHU-PL-Lab/pds-reachability"
+bug-reports: "https://github.com/JHU-PL-Lab/pds-reachability/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "base-threads"
+  "batteries" {>= "3.0"}
+  "dune" {>= "1.4"}
+  "jhupllib" {>= "0.3"}
+  "ocaml-monadic" {>= "0.4.1"}
+  "ocamlbuild" {build}
+  "ocamlfind" {build}
```

Comment:
Are you actually using ocamlbuild and ocamlfind? The sources seem to just use dune

Reply:
You're correct: these dependencies are stale.

Reply:
it seems that the ocamlbuild & ocamlfind are still there

Reply:
Sorry about that!  I'm unfamiliar with these workflows.  I've updated the PR.